docs/ca/dev/configure-data-import-from-an-s3-bucket.md

```diff
@@ -657,4 +96,28 @@ $config[FileSystemConstants::FILESYSTEM_SERVICE]['s3-import'] = [
 ];
 ```
 
+## 4. Configure the data import
+
+Adjust the `DataImportConfig` class to enable the data import from the external source.
+
+**src/Pyz/Zed/DataImport/DataImportConfig.php**
+```php
+namespace Pyz\Zed\DataImport;
+
+use Spryker\Zed\DataImport\DataImportConfig as SprykerDataImportConfig;
+
+class DataImportConfig extends SprykerDataImportConfig
+{
+    /**
+     * @return bool
+     */
+    public function isDataImportFromOtherSourceEnabled(): bool
+    {
+        return true;
+    }
+}
+```
+
+
+
```

Comment:
Could you add, please, an example of data inmport configuration .yml file as we discussed?

Reply:
Do you mean this part https://github.com/spryker/spryker-docs/pull/2787/files#diff-af56f3bb7b7cace2d291f88754064a2a9595d7ffc7dea180a20ef4e4d56561d5R38 ?